target/linux/ar71xx/files/arch/mips/ath79/Kconfig.openwrt

```diff
@@ -1998,6 +1998,15 @@ config ATH79_MACH_CF_E320N_V2
 	select ATH79_DEV_USB
 	select ATH79_DEV_WMAC
 
+config ATH79_MACH_CF_E355AC
+      bool "COMFAST CF-E355AC board support"
+      select SOC_QCA953X
+      select ATH79_DEV_ETH
+      select ATH79_DEV_GPIO_BUTTONS
+      select ATH79_DEV_LEDS_GPIO
+      select ATH79_DEV_M25P80
+
+
```

Comment:
Drop this redundant empty line, please.

Also, your are missing here:
- `select ATH79_DEV_AP9X_PCI if PCI`
- `select ATH79_DEV_AP9X_PCI if PCI`